src/Calendars/HijriCalendar.php

```diff
@@ -0,0 +1,375 @@
+<?php
+
+namespace Spatie\Holidays\Calendars;
+
+use Carbon\CarbonImmutable;
+use RuntimeException;
+
+trait HijriCalendar
```

Comment:
As I understand correctly, these dates are not the same for all Islamic countries?

In that case, this class can probably not be reused, so I think it's better to move them to the Tunisia class.

Reply:
the difference generally in one day, but i changed it to Tunisia only and i removed the trait